js/main.js

```diff
@@ -16,23 +16,25 @@ class Main {
    */
   constructor() {
     // Initiate variables
-    this.chatTextBoxDiv = document.getElementById('chatbox-div');
-    this.chatTextBox = document.getElementById('generated-chat');
+    this.conversationEle = document.getElementById('conversation');
     this.inputText = document.getElementById('input-text');
+    this.modelLoadAnimEle = document.getElementById('model-load-anim');
     this.inputText.onkeyup = (evt) => {
         evt.preventDefault();
         if (evt.keyCode === 13) {
             this.sendChat();
         } 
     }
     this.chatContent = [];
-    this.temperature = 0
-    this.temperatureSlider = document.getElementById('slider-range')
-    this.temperatureDisplay = document.getElementById('temperature-display')
+    this.temperature = 0;
+    this.temperatureSlider = document.getElementById('slider-range');
+    this.temperatureDisplay = document.getElementById('temperature-display');
     this.temperatureSlider.oninput = (evt) => {
-        this.temperature = evt.target.value/100.
-        this.temperatureDisplay.innerHTML = "Temperature: " + this.temperature;
+        this.temperature = parseFloat(evt.target.value);
```

Comment:
Didnt read the code much but I'm assuming evt.target.value is already from 0 to 1?